### PR TITLE
Modified footer

### PR DESCRIPTION
**Pull request by abhisheknaiidu**
I think it looks fine now :)

in2

### CODE REVIEW

File: views/partials/footer.ejs [hunk 1]
```diff
@@ -10,4 +12,5 @@
     <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
     <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js" integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>
 </body>
-</html>
\ No newline at end of file
+</html>
+</footer>
```
Comment (thebishaldeb) [inline views/partials/footer.ejs]: no tag should come below `html`

**Review by abhisheknaiidu — COMMENTED** [review]
@thebishaldeb  I have made changes.

File: views/partials/footer.ejs (hunk 1 above)
Comment (abhisheknaiidu) [reply views/partials/footer.ejs]: I have made changes.

File: views/partials/footer.ejs [hunk 2]
```diff
@@ -10,4 +12,5 @@
     <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
     <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js" integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>
 </body>
-</html>
\ No newline at end of file
+</footer>
```
Comment (thebishaldeb) [inline views/partials/footer.ejs]: Please follow the order. You see the `footer` is a child of the `body` tag. Please follow the order. I thought you'd see it when you correct the `html` tag. The site may crash at some point if the order of parent child component is not followed.

**Review by abhisheknaiidu — COMMENTED** [review]
@thebishaldeb See this commit, I have those changes

File: views/partials/footer.ejs (hunk 2 above)
Comment (abhisheknaiidu) [reply views/partials/footer.ejs]: @thebishaldeb I guess you are seeing my previous commits, I have modified it in my latest commit.

Comment (thebishaldeb) [inline views/partials/footer.ejs]: first `footer` tag closes and then `body`, not vice versa.

**Comment by thebishaldeb** [comment]
> @thebishaldeb See this commit, I have those changes

Which one? File changes isn't visible.

File: views/partials/footer.ejs (hunk 2 above)
Comment (abhisheknaiidu) [reply views/partials/footer.ejs]: @thebishaldeb see my latest commit. I guess commit no. 4

Comment (abhisheknaiidu) [reply views/partials/footer.ejs]: commit : 6b3545a
https://github.com/NJACKWinterOfCode/Instances/pull/42/commits/6b3545a17d7fcf143b18384a920f0066726071de

**Pull request closed without merging**